### ISSUE TITLE
Dark mode for settings page

### ISSUE THREAD

**Issue by XDelta** [Mephiles/torntools_extension#271]
**Describe the solution you'd like**
Theming options for settings page? At least a dark mode/option, the pure white is very bright.

**Comment by DeKleineKobini** [comment]
Honestly, I don't think it's worth it to theme the settings page.

**Comment by Mephiles** [comment]
How often do you visit the settings page? If often, then why?

**Comment by XDelta** [comment]
It can be a single check box for a dark appearance. Using 'theming' implies too much I suppose. As for checking settings - Reading latest change log, going to target list. Not often but would be a 'nice to have'.

**Comment by XDelta** [comment]
Adding the following css as a toggle is a quick solution (although it'll affect images in settings),
`
background: #212121;
filter: invert() hue-rotate(180deg) brightness(105%) contrast(105%);
`
Examples:
![image](https://user-images.githubusercontent.com/7883807/96371519-eb537500-1116-11eb-8ccf-db6621e663d3.png)
![image](https://user-images.githubusercontent.com/7883807/96371524-f0b0bf80-1116-11eb-8de1-5f26cd910ec6.png)
![image](https://user-images.githubusercontent.com/7883807/96371531-f8706400-1116-11eb-951d-16144738511e.png)

**Comment by raxty** [comment]
We could add this as dark mode (also in the popup?), and for the images one can just invert again e.g.:

```css
body.dark {
  background: #212121;
  filter: invert() hue-rotate(180deg) brightness(105%) contrast(105%);
}

body.dark .icon {
  filter: invert() hue-rotate(180deg) brightness(105%) contrast(105%);
}
```

**Comment by DeKleineKobini** [comment]
This will not be added in the current major version (5.x), and is already implemented for the next big version (6.0).
